spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/data/TestSparkOrcReadMetadataColumns.java

```diff
@@ -123,7 +124,7 @@ public void writeFile() throws IOException {
         // write in such a way that the file contains 10 stripes each with 100 rows
         .set("iceberg.orc.vectorbatch.size", "100")
         .set(OrcConf.ROWS_BETWEEN_CHECKS.getAttribute(), "100")
-        .set(OrcConf.STRIPE_SIZE.getAttribute(), "1")
```

Comment:
So looks like the default value of table property `write.orc.stripe-size-bytes` will override the set value from original ORC config key (`orc.stripe.size` ) ?

Reply:
Yes, using the original ORC config key directly here is invalid and will be overwritten by the table property or by the table property default value.

Reply:
If the standard iceberg table properties is not found, do we feedback to try the origin ORC key?

Reply:
I think the table property should override the Hadoop configuration property. We should probably also support the old `iceberg.orc.vectorbatch.size` property passed in like this. I think the table property should override it.